model/src/main/proto/profile.proto

```diff
@@ -82,9 +82,6 @@ message Profile {
 
   // Determines if the in-exploration continue button animation has been seen by the user or not.
   bool is_continue_button_animation_seen = 15;
-
-  // Represents user selected content-language.
-  OppiaLanguage oppia_language = 17;
```

Comment:
This needs to be replaced with a 'reserved' explaining what the previous value was (see elsewhere in the codebase on how we document deprecated proto fields).

Reply:
updated.